### ISSUE TITLE
install sense2vec in Windows

### ISSUE THREAD

**Issue by leo-gan** [explosion/sense2vec#62]
`(base) C:\WINDOWS\system32>pip install sense2vec==1.0.0a0
Collecting sense2vec==1.0.0a0
  Downloading https://files.pythonhosted.org/packages/28/4a/a1d9a28545adc839789c1442e7314cb0c70b8657a885f9e5b287fade7814/sense2vec-1.0.0a0.tar.gz (311kB)
    100% |████████████████████████████████| 317kB 1.5MB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\...\AppData\Local\Temp\pip-install-5_i77ipi\sense2vec\setup.py", line 169, in <module>
        setup_package()
      File "C:\Users\...\AppData\Local\Temp\pip-install-5_i77ipi\sense2vec\setup.py", line 107, in setup_package
        readme = f.read()
      File "c:\users\...\anaconda3\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 13089: character maps to <undefined>

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\...\AppData\Local\Temp\pip-install-5_i77ipi\sense2vec\`

**Comment by thecodist** [comment]
FYI - I was just about to report the exact same thing and then noticed it's already been reported. :-)

**Comment by DeepakSaini119** [comment]
Getting the same error

**Comment by frozamilan1899** [comment]
Same error as I met. 
Anyone could sovle this issue?

**Comment by thecodist** [comment]
I believe the solution for me finally was to bring it down manually and edit the line 107 in setup.py removing "io" from "io.open". It's been a while and I haven't tried it lately so no guarantees. I'd be interested to know if this works for you.
From:
with io.open(os.path.join(root, src_path, 'about.py'), encoding='utf8') as f:
To:
with open(os.path.join(root, src_path, 'about.py'), encoding="utf8") as f:

**Comment by thecodist** [comment]
Or maybe line 103. I can't remember for sure.

**Comment by Barachia** [comment]
To give an overview of steps what I did to solve the problem:

1. Download the [source code](https://github.com/explosion/sense2vec/archive/v1.0.0a0.zip) and unzip it.
2. Edit the `..\sense2vec-1.0.0.a0\setup.py` with a text editor, with the following two changes: add `encoding="utf8"` in line 102 and line 106. The second edit is probably not necessary, but for consistency I changed this as well. So:
` with open(os.path.join(root, src_path, 'about.py')) as f:`
becomes
` with open(os.path.join(root, src_path, 'about.py'), encoding="utf8") as f:`
and
`with open(os.path.join(root, 'README.rst')) as f:`
becomes
`with open(os.path.join(root, 'README.rst'), encoding="utf8") as f:`

3. Open op a terminal (make sure for using the right environment) 
4. If you don't have cython yet, use `pip install cython` (I used cython-0.29.7)
5. Run `pip install "<location_of_source_download>/sense2vec-1.0.0a0" `
6. Run `pip install -r "<location_of_source_download>/requirements.txt"`
7. Now sense2vec is installed, we need to add the spacy-services for sense2vec as well. Go to the folder (something like `...\spacy-services\sense2vec\`) where you have the spacy-services and run `pip install -r requirements.txt`
7. To test if your server is working, now run `python app.py` 

After running it, I do get these warnings:
`
C:\Anaconda3\envs\sense2vec\lib\importlib\_bootstrap.py:219: RuntimeWarning: cymem.cymem.Pool size changed, may indicate binary incompatibility. Expected 48 from C header, got 64 from PyObject return f(*args, **kwds)
`
`
C:\Anaconda3\envs\sense2vec\lib\importlib\_bootstrap.py:219: RuntimeWarning: cymem.cymem.Address size changed, may indicate binary incompatibility. Expected 24 from C header, got 40 from PyObject
  return f(*args, **kwds)
`
